x/clock/abci.go

```diff
@@ -28,12 +28,12 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 			continue
 		}
 
-		_, err = k.GetContractKeeper().Sudo(ctx, contract, message)
+		childCtx := ctx.WithGasMeter(sdk.NewGasMeter(types.EndBlockContractGasLimit))
```

Comment:
I like this more as a safety check since it is in the endblock anyways. (1bn gas was used by terra pre collapse, and in my testing a VPS node can support it in the endblocker no problem)

Reply:
May want to move this to a gov param

Reply:
Added to gov params 83d51d2dba40ca73bfd82c56c4a77ecb6075bf8f